src/renderer/versions.ts

```diff
@@ -179,6 +179,22 @@ export function addLocalVersion(input: Version): Array<Version> {
   return versions;
 }
 
+/**
+ * Check to see if local version has already been added
+ * for the current folder path
+ *
+ * @param {Version} input
+ * @returns {Version | undefined}
+ */
+export function getExistingLocalVersion(
```

Comment:
Another naming suggestion for consistency with other utils like `getElectronNameForPlatform()`, `getLocalTypePathForVersion()`, etc

```suggestion
export function getLocalVersionForPath(
```